### PR TITLE
Workspace detail forms

### PR DESCRIPTION
**Pull request by jtomeck**
Drop forms inside modals and match styling of workspace detail page.

### CODE REVIEW

File: client/src/views/WorkspaceDetail.vue [hunk 1]
```diff
@@ -85,22 +85,72 @@
             color="transparent"
             flat
           >
-            <v-card-title>
+            <v-card-title class="pb-2">
               <h2>Create Tables</h2>
```
Comment (waxlamp) [inline client/src/views/WorkspaceDetail.vue]: Change this to just "Tables".

File: client/src/views/WorkspaceDetail.vue [hunk 2]
```diff
@@ -122,39 +172,92 @@
...
-                  <v-select
-                    v-model="graphEdgeTable"
-                    :items="edgeTables"
-                    solo
-                  />
-                </v-flex>
-                <v-btn :disabled="graphCreateDisabled" @click="createGraph">create graph</v-btn>
-              </v-layout>
+          <v-card-title class="pb-2">
+            <h2>Create Graphs</h2>
```
Comment (waxlamp) [inline client/src/views/WorkspaceDetail.vue]: Change this to just "Graphs".

File: client/src/views/WorkspaceDetail.vue [hunk 3]
```diff
@@ -122,39 +172,92 @@
...
+                  </v-card-actions>
+                </v-card>
+              </v-card>
+            </v-dialog>
+          </v-card-title>
+
+            <v-card-text class="pt-0">
+              <v-divider />
+              <div class="content pt-3">
+                <p class="grey--text text--darken-1">Graphs info Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus non est et nisl egestas molestie. Morbi congue varius ipsum.</p>
```
Comment (waxlamp) [inline client/src/views/WorkspaceDetail.vue]: How hard is it to move this inside a little question mark popover-type thing next to the "Tables" title?

I will also need to send you some copy for this (and the other lorem ipsum block).

File: client/src/views/WorkspaceDetail.vue [hunk 4]
```diff
@@ -85,22 +85,72 @@
...
-              <v-layout row wrap>
-                <v-flex>
-                  <file-input @handle-file-input="handleFileInput" v-bind:types="fileTypes"/>
-                </v-flex>
-              </v-layout>
-              <v-btn :disabled="tableCreateDisabled" @click="createTable">create table</v-btn>
+            <v-card-text class="pt-0">
+              <v-divider />
+              <div class="content pt-3">
+                <p class="grey--text text--darken-1">Tables info Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus non est et nisl egestas molestie. Morbi congue varius ipsum.</p>
```
Comment (waxlamp) [inline client/src/views/WorkspaceDetail.vue]: I need to send you copy for this.

File: client/src/views/WorkspaceDetail.vue (hunk 3 above)
Comment (jtomeck) [reply client/src/views/WorkspaceDetail.vue]: @waxlamp - it doesn't look great with the design, imo. I have a couple ideas if you want to discuss at my desk at some point (and I can show you what I'm talking about).

File: client/src/views/WorkspaceDetail.vue (hunk 4 above)
Comment (jtomeck) [reply client/src/views/WorkspaceDetail.vue]: Based on our conversation, we're disregarding this for now.

File: client/src/views/WorkspaceDetail.vue (hunk 3 above)
Comment (jtomeck) [reply client/src/views/WorkspaceDetail.vue]: Based on our conversation, we're disregarding this for now.

**Comment by waxlamp** [comment]
@jtomeck, just FYI I rebased this branch off of master, and force-pushed. It still has the exact commits you made (plus one I just added to drop the "Your" from the item panel titles).

**Pull request merged**